c/CHANGELOG.rst

```diff
@@ -1,5 +1,5 @@
 ----------------------
-[0.99.14] - 2021-0X-XX
+[0.99.14] - 2021-09-03
 ----------------------
 
```

Comment:
We've got a bunch of breaking changes here, don't we?

Reply:
Yep, just rebased to pull that in.